### PR TITLE
app-emulation/rkt: bump to 0.8.1 release

### PR DESCRIPTION
**Pull request by jonboulle**
fixes #1516

### CODE REVIEW

**Comment by crawford** [comment]
I believe the live ebuild still refers to the service and timer units.

**Comment by jonboulle** [comment]
@crawford updated

File: app-emulation/rkt/rkt-9999.ebuild [hunk 1]
```diff
@@ -14,16 +14,16 @@ CROS_WORKON_PROJECT="coreos/rkt"
 CROS_WORKON_LOCALNAME="rkt"
 CROS_WORKON_REPO="git://github.com"
 
+
 if [[ "${PV}" == "9999" ]]; then
 	KEYWORDS="~amd64"
-	PXE_VERSION="738.1.0"
-
-elif [[ "${PV}" == "0.7.0" ]]; then
+elif [[ "${PV}" == "0.8.1" ]]; then
```
Comment (marineam) [inline app-emulation/rkt/rkt-9999.ebuild]: We can probably just switch this to an else and add a `# 0.8.1` comment which is our more standard pattern.

**Comment by marineam** [comment]
Go ahead and drop `rkt-0.5.5-r1.ebuild` which also refers to the old units. we don't need that ebuild.

File: app-emulation/rkt/rkt-9999.ebuild (hunk 1 above)
Comment (jonboulle) [inline app-emulation/rkt/rkt-9999.ebuild]: up day ted

**Comment by jonboulle** [comment]
anything else

**Comment by marineam** [comment]
LGTM!

**Pull request merged**